jf_agent/main.py

```diff
@@ -247,8 +247,8 @@ def obtain_config(args):
     jira_exclude_project_categories = set(jira_config.get('exclude_project_categories', []))
     jira_issue_jql = jira_config.get('issue_jql', '')
 
-    git_config = yaml_config.get('git', yaml_config.get('bitbucket', {}))
-    git_provider = git_config.get('provider', 'bitbucket_server')
+    git_config = yaml_config.get('git', {})
+    git_provider = git_config.get('provider', None)
     git_url = git_config.get('url', None)
```

Comment:
These defaults were in there to be backwards-compatible with old YML format.  It's possible that parallel etc are still using that old YML config file, so we may want to leave the defaults in.